### PR TITLE
source-airbyte contribution from johnwasserman

### PR DESCRIPTION
**Pull request by johnwasserman**
Auto-generated PR by the Connector Builder for source-airbyte

### CODE REVIEW

**Comment by postamar** [comment]
This is so meta.

File: airbyte-integrations/connectors/source-airbyte/metadata.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+metadataSpecVersion: "1.0"
+data:
+  allowedHosts:
+    hosts:
+      - "*"
```
Comment (natikgadzhi) [inline airbyte-integrations/connectors/source-airbyte/metadata.yaml:5]:
```suggestion
      - api.airbyte.com
```

File: docs/integrations/sources/airbyte.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+# Airbyte
+
+
```
Comment (natikgadzhi) [inline docs/integrations/sources/airbyte.md:3]: This source allows you to sync up data about your Airbyte Cloud workspaces. [Take a look at this guide](https://docs.airbyte.com/using-airbyte/configuring-api-access) to setup API access tokens.

Comment (natikgadzhi) [inline docs/integrations/sources/airbyte.md:3]:
```suggestion
This source allows you to sync up data about your Airbyte Cloud workspaces. [Take a look at this guide](https://docs.airbyte.com/using-airbyte/configuring-api-access) to setup API access tokens.
```

**Comment by natikgadzhi** [comment]
@johnwasserman 👏🏼

**Pull request merged**